### ISSUE TITLE
About the pretraining data of Chinese

### ISSUE THREAD

**Issue by zjh-nudger** [google-research/bert#181]
I am now pretraining Chinese bert model from scratch. I downloaded the Chinese wiki data and clean up it. Then, I created data using `create_pretraining_data.py` and found that only 15M instances were be writed to tfrecord data. Is it all right?

**Comment by FrankWork** [comment]
@zjh-nudger  How long does it take to finish training?

**Comment by Sara-HY** [comment]
@zjh-nudger  Hi, I am pretraining Chinese bert model too. But the loss of the model cannot drop down when it get 7. How about you loss?
